css/style.css

```diff
@@ -342,3 +345,143 @@ hr {
   background-color: #6070ff;
   color: #fff;
 }
+
+@media only screen and (min-width: 175px) and (max-width: 420px) and (orientation: portrait) {
+  body {
+    background-image: url("../img/header-background-mobile.png");
+    background-repeat: no-repeat;
+    background-position: 0 0 0 0;
+  }
+
+  .nav-container {
+    display: none;
+  }
+
+  .nav-space {
+    display: none;
+  }
+}
+
+@media (min-width: 421px) {
+  body {
+    background-image: url("../img/header2.png");
+    background-repeat: no-repeat;
+    background-position: 0 0 0 0;
+  }
+
+  header {
+    margin: 1.5625rem 9.625rem;
+    padding: 0;
+  }
+
+  .top-bars {
+    display: none;
+  }
+
+  .nav-container {
+    display: flex;
+    flex-grow: 1;
+  }
+
+  .nav-space {
+    display: flex;
+    flex-grow: 2;
+  }
+
+  a {
+    text-decoration: none;
+    color: #6070ff;
+  }
+
+  a:hover {
+    text-decoration: underline;
+    text-decoration-color: #6070ff;
+  }
+
+  .nav-link {
+    width: 100%;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: space-between;
+    color: #344563;
+    font-size: 1rem;
+    line-height: 1.25rem;
+    font-weight: 500;
+  }
+
+  .mx-container {
+    margin: 18rem 16.25rem 16rem 16.25rem;
+  }
+
+  .card {
+    width: 100;
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-end;
+  }
+
+  .card-inverse {
+    display: flex;
+    flex-direction: row-reverse;
+  }
+
+  .card-image{
+    width: fit-content;
+  }
+
+  .card-text-container {
+    margin-left: 2.875rem;
+  }
+
+  .card-inverse-text {
+    margin-right: 3rem;
+  }
+
+  .form-container {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+  }
+
+  .form-input {
+    width: 45%;
+  }
+
+  .contact {
+    background-image: url("../img/contact-l.png");
+    background-size: cover;
+    background-repeat: no-repeat;
+  }
+
+  .btn-wrapper {
+    justify-content: center;
+  }
+
+  .about-container {
+    background-color: #fff;
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    padding: 8.875rem;
+    margin-top: 8.875rem;
+  }
```

Comment:
- [ ] The about-container has a border radius on the right corner in the Figma template, please 🙏🏽 fix it to match the original design.

| Figma template | Your design |
|---|---|
| ![image](https://user-images.githubusercontent.com/58642949/222601350-4765064c-0f53-4c23-acd5-c4d14aa5af68.png) | ![image](https://user-images.githubusercontent.com/58642949/222601416-5f8b2570-a868-4078-bc5f-df37dbb6acf3.png) |